# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1426699525 14400
#      Wed Mar 18 13:25:25 2015 -0400
# Node ID 7f44ced38646e28116466e22cfb370201c7270aa
# Parent  749104c1bb1ba737acbc0d860559186b0d9df251
* contributors.in: One more update.

diff --git a/doc/interpreter/contributors.in b/doc/interpreter/contributors.in
--- a/doc/interpreter/contributors.in
+++ b/doc/interpreter/contributors.in
@@ -204,16 +204,17 @@ Laurent Mazet
 G. D. McBain
 Ronald van der Meer
 Júlio Hoffimann Mendes
 Ed Meyer
 Thorsten Meyer
 Stefan Miereis
 Petr Mikulik
 Mike Miller
+Serviscope Minor
 Stefan Monnier
 Antoine Moreau
 Kai P. Mueller
 Hannes Müller
 Victor Munoz
 PrasannaKumar Muralidharan
 Iain Murray
 Carmen Navarrete
